pkgs/development/tools/protoc-gen-dart/update.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p yq ripgrep common-updater-scripts dart
+
+set -xeu -o pipefail
+
+PACKAGE_DIR="$(realpath "$(dirname "$0")")"
+cd "$PACKAGE_DIR/.."
+while ! test -f default.nix; do cd .. ; done
```

Comment:
```suggestion
while ! test -f flake.nix; do cd .. ; done
```

Reply:
I actually avoided using flake.nix here, since it's still a experimental feature.
I updated the logic to be more in line with what other update scripts do.